src/plugins/unified_search/public/filter_bar/filter_item/filter_item.tsx

```diff
@@ -78,7 +78,7 @@ export function FilterItem(props: FilterItemProps) {
   useEffect(() => {
     const index = props.filter.meta.index;
     let isSubscribed = true;
-    if (index) {
+    if (index && !index.startsWith('local')) {
```

Comment:
cc @lukasolson as we now can't recognize local data views from the id this check is no longer possible. Any idea how we could work around that ?

Reply:
I'm checking if we can delay this check until the page has fully loaded ....